mastering-plone/dexterity_2.rst

```diff
@@ -152,7 +152,7 @@ Upgrade steps
 When projects evolve you sometimes want to modify various things while the site is already up and brimming with content and users.
 Upgrade steps are pieces of code that run when upgrading from one version of an add-on to a newer one.
 They can do just about anything.
-We will use an upgrade-step to enable the new behavior instead of reinstalling the addon.
+We will use an upgrade step to enable the new behavior instead of reinstalling the addon.
```

Comment:
s/addon/add-on throughout doc.

Reply:
Steve, I'm tweaking things as I find them. I am not making wholesale changes to this material. Let's go with progressive improvements and I will apply regexps consistently to the whole thing when I am home (am teaching this class today too).